### PR TITLE
episode 35 GeeksBlaBla nfaihi/ep35

### PR DESCRIPTION
**Pull request by nfaihi**
Add episode 35 of GeeksBlaBla about DevC Community

### CODE REVIEW

File: blablas/ep35/index.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+tags: ["GeeksBlabla", "DevC", "Community", "Facebook"]
+isNext: false
+video: "2982251615130868"
+published: true
+featured: false
+url: ""
+podcastUrl: https://web.facebook.com/youssouf.elazizi/videos/2982251615130868
+---
+
+In this special episode of GeeksBlabla, some of its core team members discussing what is DevC community, how it starts and what is the relationship between it and Facebook. In the end, the members start revealing some of their secrets and memories they had in the community.
```
Comment (aboullaite) [inline blablas/ep35/index.md:15]: Can you please change:
+ Discussing --> discussed
+ How it starts --> how is started
+ What is the relationship... -> how facebook supports devc

Comment (aboullaite) [inline blablas/ep35/index.md:15]: In the end --> At the end
Start reveiling --> reveiled

**Comment by aboullaite** [comment]
Thanks a lot @nfaihi for your contribution, I requested some changes

**Comment by yjose** [comment]
Title without GeeksBlabla 🙏

**Comment by nfaihi** [comment]
Requested changes are made. Thanks @aboullaite and @yjose for the help 🙏

**Comment by aboullaite** [comment]
@nfaihi one last change request please (last line `their secrets --> the secrets`). also could you please squash the commits to one. Thank you

**Comment by nfaihi** [comment]
@aboullaite could you check now please?

**Comment by aboullaite** [comment]
> @aboullaite could you check now please?
Bravo! Amazing Job 👍 @yjose LGTM

**Comment by yjose** [comment]
@aboullaite Merged, Thanks @nfaihi  for your contribution,  @all-contributors please add @nfaihi for content

**Comment by nfaihi** [comment]
> 
> 
> > @aboullaite could you check now please?
> 
> @bondif Bravo! Amazing Job 👍 @yjose LGTM

@bondif ??

**Comment by nfaihi** [comment]
> 
> 
> @aboullaite Merged, Thanks @nfaihi for your contribution, @all-contributors please add @nfaihi for content

Thank you so much @yjose and @aboullaite . It's a pleasure for me 🙏

**Pull request merged**